forms/js/index.js

```diff
@@ -0,0 +1,84 @@
+// ## Формы
+
+// - Добавьте в документ несколько форм (не менее 10), каждая форма должна иметь уникальное имя 
+// (задать с помощью соответствующего аттрибута).
+
+
+// - Отобразите на странице имена всех четных форм через запятую.
+
+
+// - Добавьте в форму не менее 2 полей (разных видов и разное количество).
+
+
+// - Добавьте в каждую форму кнопку “Показать имя формы”. 
+// При клике на кнопку показать в стандартном всплываюдщем окне текст, отображаемый на кнопке
+
+
+// - Добавьте в каждую форму кнопку “Принадлежность”. При клике на эту кнопку показать в всплывающем окне id формы, 
+// которой принадлежит кнопка.
+
+
+// - Создайте функцию для сброса полей формы. Добавьте к каждой форме кнопку “Сбросить”. 
+// Эта кнопка сбрасывает все значения той формы, на которую кликнул пользователь.
+
+
+// - Добавьте в каждую форму кнопку “Показать количество полей”.
+//  При клике на ней показывается высплывающее окно с кол-во полей в форме.
+
+const getElFromDom = document.querySelectorAll('form')
+console.log(getElFromDom)
+
+const evenForm = function(arr) {
+    let enevArray =[];
+    for(let i = 0; i < arr.length; i+=2) {
```

Comment:
чтобы не выйти за пределы массива, нужно ограничить i
```suggestion
    for(let i = 0; i < arr.length - 2; i+=2) {
```